puremagic/main.py

```diff
@@ -398,10 +406,34 @@ def single_deep_scan(
             result = sndhdr_scanner.main(filename, head, foot)
             if result and result.confidence > confidence:
                 return result
+        case (
+            mpeg_audio_scanner.mp3_id3_match_bytes
+            | mpeg_audio_scanner.raw_mp3_match_bytes
+            | mpeg_audio_scanner.fffe_match_bytes
+            | mpeg_audio_scanner.ffff_match_bytes
+            | mpeg_audio_scanner.fffc_match_bytes
+            | mpeg_audio_scanner.fffd_match_bytes
+            | mpeg_audio_scanner.fffa_match_bytes
+            | mpeg_audio_scanner.fff6_match_bytes
+            | mpeg_audio_scanner.fff7_match_bytes
+            | mpeg_audio_scanner.fff4_match_bytes
+            | mpeg_audio_scanner.fff5_match_bytes
+            | mpeg_audio_scanner.fff2_match_bytes
+            | mpeg_audio_scanner.fff3_match_bytes
+            | mpeg_audio_scanner.ffe6_match_bytes
+            | mpeg_audio_scanner.ffe7_match_bytes
+            | mpeg_audio_scanner.ffe4_match_bytes
+            | mpeg_audio_scanner.ffe5_match_bytes
+            | mpeg_audio_scanner.ffe2_match_bytes
+            | mpeg_audio_scanner.ffe3_match_bytes
+        ):
```

Comment:
```suggestion
        case mpeg_bytes if mpeg_bytes in mpeg_audio_scanner.mpeg_audio_signatures
        ):
```

Would just put those all in a single array